### PR TITLE
Added test for checking if settings model contains default value

### PR DESCRIPTION
**Pull request by andr317c**
Added tests for checking if the block list editor settings model contains the defined default value.

Also did some cleaning up on blockListEditorContent file

The test was created so we would be able to see if we run into this issue again: #13134

### CODE REVIEW

**Review by Zeegaan — CHANGES_REQUESTED** [review]
Few small things 😁

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts [hunk 1]
```diff
@@ -89,13 +89,43 @@ test.describe('BlockListEditorContent', () => {
...
     return element;
   }
 
+  async function createSliderWithDefaultValue(umbracoApi, sliderName: string, defaultValue: number) {
+    const slider = new SliderDataTypeBuilder()
+      .withName(sliderName)
+      .withInitialLowerValue(defaultValue)
+      .build();
+    return await umbracoApi.dataTypes.save(slider);
+  }
```
[the remark this answers is not in the document]
Comment (andr317c) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts]: Fixed 💪

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts [hunk 2]
```diff
@@ -113,25 +143,25 @@ test.describe('BlockListEditorContent', () => {
...
     await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.saveAndPublish));
 
     // Assert
     await umbracoUi.isSuccessNotificationVisible();
-    
+
     // Checks if the content was created
     await expect(page.locator('.umb-block-list__block--view')).toHaveCount(1);
-    await expect(page.locator('.umb-block-list__block--view').nth(0)).toHaveText(elementName);
+    await expect(page.locator('.umb-block-list__block--view').nth(0)).toHaveText(contentElementName);
```
[the remark this answers is not in the document]
Comment (andr317c) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts]: That's a good point, no reason to have count when we only have 1 😆

**Review by jdpnielsen — APPROVED** [review]

**Review by Zeegaan — APPROVED** [review]
LGTM 👍

**Pull request merged**